src/ServicePulse.Host/vue/src/composables/serviceMessageGroup.js

```diff
@@ -0,0 +1,73 @@
+import {
+  useDeleteFromServiceControl,
+  usePostToServiceControl,
+  useFetchFromServiceControl,
+} from "./serviceServiceControlUrls.js";
+
+export function useGetExceptionGroups() {
```

Comment:
I'm not sure thrilled by the naming convention. What about dropping the "use" parts?

E.g. `getExceptionGroups() {`

Reply:
me neither. I continued to follow the conventions on how it was used in other vue pages.

Reply:
In the [documentation](https://github.com/Particular/Platform/blob/main/documentation/service-pulse.md#best-practises) it even states that the best practice is to have exported function begin with "use"

Reply:
Ok, let's leave it as is for now. It feels super weird to me though